### DISCUSSION TITLE
win7编译后运行Hello2D.exe错误

### DISCUSSION THREAD

**Discussion by ybkyozy** [Tencent/tgfx#297] [Q&A] [answered]
提示:
无法启动此程序，因为计算机中丢失 api-ms-win-shcore-scaling-l1-1-1.dll。尝试重新安装该程序以解决此问题。

**Comment by domchen** [comment]
应该是Windows 7 缺少DPIAware相关的接口，我们手边都没有Windows 7 的环境可以测试，你可以试下修改这行： https://github.com/Tencent/tgfx/blob/main/win/CMakeLists.txt#L27 里的 find_library(SHCORE_LIB SHCore) 为 find_library(SHCORE_LIB user32), 重新编译看看能否运行成功。这个问题是因为Demo里调用了DPIAware，自己写一个Demo不涉及这个API的话应该也不影响渲染。

**Reply by domchen** [reply, marked as answer]
我们在 #366 里修复了大量Windows平台demo的问题，更新到最新代码，应该可以正常运行了。
